### ISSUE TITLE
[GH Request] Onboard <mrazadar, MuhammadMattiullah, NawfalAhmed>

### ISSUE THREAD

**Issue by kmedina2u** [openedx/tcril-engineering#653]
### Firm Name

2U

### User List (case sensitive!) - Please also include in ticket title

* Muhammad Raza Dar, @mrazadar
* Muhammad Matti-Ullah, @MuhammadMattiullah
* Nawfal Ahmed, @NawfalAhmed

### Start Date

6 Februrary 2023

### Urgency

Urgent

### Special Requests

Emails:

* Muhammad Raza Dar: raza.dar@arbisoft.com, mrazadar@2u.com
* Muhammad Matti-Ullah: mattiullah@arbisoft.com, mmatti-ullah@2u.com
* Nawfal Ahmed: nawfal.ahmed@arbisoft.com, nahmed1@2u.com

**Comment by kdmccormick** [comment]
@brian-smith-tcril I'm going to leave this one for you to try next week. Let me know if you run into any problems.

**Comment by brian-smith-tcril** [comment]
I can verify that all 3 users mentioned are:
- [x] Members of the [Open edX GH org](https://github.com/openedx)
- [x] Members of the [push-pull-all team](https://github.com/orgs/openedx/teams/push-pull-all/)

I have not added anyone to the CLA database on Salesforce yet, as I'm unsure which Salesforce account to use.

@mrazadar @MuhammadMattiullah @NawfalAhmed is your employer Arbisoft or 2U?

**Comment by sarina** [comment]
@kmedina2u could you please answer the above question ASAP?

**Comment by kmedina2u** [comment]
@sarina These 3 employees are under the Arbisoft CLA

**Comment by sarina** [comment]
thanks @kmedina2u - we've been operating off the assumption that someone with a 2U email address was being paid by 2U so would be under the 2U CLA. Is that not the case? If so could you please be very explicit when opening these tickets as to what CLA new employees are under?

**Comment by kmedina2u** [comment]
@sarina As far as I know, that assumption is correct when it comes to full time 2U employees. In the case of contractors (like the ones in this ticket), my understanding is that they are provided a 2U email address but are paid for their contributions by their contracting company (in this case Arbisoft). I will make a note in our documentation to confirm which CLA a user is under and include that information in the request so we can avoid this issue moving forward. Thank you for bringing that to my attention.

**Comment by sarina** [comment]
Yeah, it'd be good to clarify this once and for all @kmedina2u . My understanding (I think from @e0d?) was that since 2U pays for the contributions, 2U owns them, thus they'd be under 2U's license.

**Comment by georgebabey** [comment]
@sarina correct - 2U contractors should fall under the 2U CLA, regardless of what partner/agency they are from.

**Comment by sarina** [comment]
Thanks @georgebabey - once our access to Salesforce is restored (darn tcril -> axim cutover) we'll get this group noted properly as under the 2U CLA.

@brian-smith-tcril can I task you with finishing this up?

**Comment by brian-smith-tcril** [comment]
> @brian-smith-tcril can I task you with finishing this up?

Sure thing! I'm on it!
